UPDATING_TO_2.0.md

```diff
@@ -0,0 +1,889 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+-->
+# Updating Airflow
+
+This file documents any backwards-incompatible changes in Airflow and
+assists users migrating to a new version.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Step 1: Upgrade to Airflow 1.10.13 (a.k.a our "bridge" release)](#step-1-upgrade-to-airflow-11013-aka-our-bridge-release)
+- [Step 3: Upgrade to Python 3](#step-3-upgrade-to-python-3)
+- [Step 3: Upgrade Airflow DAGs](#step-3-upgrade-airflow-dags)
```

Comment:
The sequence numbers of the steps are a bit confusing and need to be modified.

Reply:
@yangrong688 does this look better? Not sure why it wasn't updating before.